dev/ssl_darwin.go

```diff
@@ -37,14 +38,24 @@ func TrustCert(cert string) error {
 		return keychainError
 	}
 
-	err := exec.Command("sh", "-c",
-		fmt.Sprintf(`security add-trusted-cert -d -r trustRoot -k '%s' '%s'`,
-			login, cert)).Run()
+	addTrustedCertCommand := exec.Command("sh", "-c",
+		fmt.Sprintf(`security add-trusted-cert -d -r trustRoot -k '%s' '%s'`, login, cert))
 
-	if err != nil {
+	stderr, readPipeErr := addTrustedCertCommand.StderrPipe()
+	if readPipeErr != nil {
+		return readPipeErr
+	}
```

Comment:
wanted to be able to examine the output if this fails, so takes a few extra lines to feed a pipe to the cmd before running it.